### DISCUSSION TITLE
Blog post url doesn't work any more

### DISCUSSION THREAD

**Discussion by triptych** [replit/kaboom#770] [General]
The blog post about Kaboom 3000 link doesn't work https://unpkg.com/kaboom@next/dist/kaboom.js 

https://kaboomjs.com/blog/3000 --> talks about inserting a script tag to the above link.
However, the response from unpkg:
```Cannot find package kaboom@next ```

**Comment by lajbel** [comment]
That should be fixed, anyway, you can use https://unpkg.com/kaboom@latest/dist/kaboom.js, but even better, use an specific version link
